tests/codegen/riscv-abi/riscv64-lp64d-abi.rs

```diff
@@ -4,7 +4,7 @@
 //@ only-linux
 #![crate_type = "lib"]
 
-// CHECK: define void @f_fpr_tracking(double %0, double %1, double %2, double %3, double %4, double %5, double %6, double %7, i8 zeroext %i)
+// CHECK: define void @f_fpr_tracking(double %0, double %1, double %2, double %3, double %4, double %5, double %6, double %7, i8 noundef zeroext %i) unnamed_addr #0
```

Comment:
`unnamed_addr #0` isn't relevant to what this is testing, you should omit it (same everywhere else in this PR)

Reply:
That was something I was curious about! I'll yoink it! Thanks!